package.json

```diff
@@ -175,6 +175,7 @@
     "**/@babel/types": "^7.10.2",
     "**/cross-fetch": "^3.0.5",
     "**/dot-prop": "^5.2.0",
+    "**/react-is": "^16.13.1",
```

Comment:
This avoids pulling in 16 and 17. But I haven't actually verified if `yarn` or `yarn deduplicate` are that aggressive so I'll catch up on that.